skbio/stats/distance/_base.py

```diff
@@ -487,6 +487,46 @@ def to_data_frame(self):
         """
         return pd.DataFrame(data=self.data, index=self.ids, columns=self.ids)
 
+    @experimental(as_of="0.5.0")
+    def to_series(self):
+        """Create a ``pandas.Series`` from this ``DissimilarityMatrix``.
+        Returns
+        -------
+        pd.Series
+            ``pd.Series`` with IDs on index.
+
+        Examples
+        --------
+        >>> from skbio import DistanceMatrix
+        >>> dm = DistanceMatrix([[0, 1, 2],[1,0,3],[2,3,0]], ids=['a','b','c'])
+        >>> df = dm.to_data_frame()
+        >>> df
+             a    b    c
+        a  0.0  1.0  2.0
+        b  1.0  0.0  3.0
+        c  2.0  3.0  0.0
+<<<<<<< HEAD
+        
+=======
+
+>>>>>>> 49864617eadb042c3f5c319c2a30694bb483e3f0
+        >>> df.to_series()
```

Comment:
Should be dm.to_series() right?  Also don't forget the to resolve merge conflicts.